segmentstore/storage/src/main/java/io/pravega/segmentstore/storage/chunklayer/AsyncBaseChunkStorage.java

```diff
@@ -712,7 +711,7 @@ protected void checkChunkName(String chunkName) {
     /**
      * Context data used to capture operation runtime information.
      */
-    static class OperationContext {
+    protected static class OperationContext {
```

Comment:
can we add a comment that explains that this class is intended to be used by subclasses and how ?